File: src/main/java/com/zippeykeys/praisebe/deity/ModAffinities.java
package com.zippeykeys.praisebe.deity;

import lombok.experimental.UtilityClass;


@UtilityClass
public class ModAffinities{
    public static final Affinity APOLLO_GRANNUS = Affinity.builder() //
      .name("apollogrannus") //
      .primaryDeity(ModDeities.Celestial.APOLLO) //
      .secondaryDeity(ModDeities.Terrestrial.GRANNUS) //
      .build();

    public static final Affinity AMUN_RA = Affinity.builder() //
      .name("amunra") //
      .primaryDeity(ModDeities.Celestial.RA) //
      .secondaryDeity(ModDeities.Celestial.AMUN) //
      .build();

    public static final Affinity ZEUS_AMMON = Affinity.builder() //
      .name("zeusammon"). //
      primaryDeity(ModDeities.Celestial.ZEUS) //
      .secondaryDeity(ModDeities.Celestial.AMUN) //
      .build();
}

File: src/main/kotlin/com/zippeykeys/praisebe/deity/ModAffinities.kt
package com.zippeykeys.praisebe.deity

import com.zippeykeys.praisebe.deity.ModDeities.Celestial.AMUN
import com.zippeykeys.praisebe.deity.ModDeities.Celestial.APOLLO
import com.zippeykeys.praisebe.deity.ModDeities.Celestial.RA
import com.zippeykeys.praisebe.deity.ModDeities.Celestial.ZEUS
import com.zippeykeys.praisebe.deity.ModDeities.Terrestrial.GRANNUS

object ModAffinities {
    val APOLLO_GRANNUS = Affinity(
            name = "apollogrannus",
            primaryDeity = APOLLO,
            secondaryDeity = GRANNUS
    )

    val AMUN_RA = Affinity(
            name = "amunra",
            primaryDeity = RA,
            secondaryDeity = AMUN
    )

    val ZEUS_AMMON = Affinity(
            name = "zeusammon",
            primaryDeity = ZEUS,
            secondaryDeity = AMUN
    )
}